websockets/uri.py

```diff
@@ -52,4 +54,7 @@ def parse_uri(uri):
     resource_name = uri.path or '/'
     if uri.query:
         resource_name += '?' + uri.query
-    return WebSocketURI(secure, host, port, resource_name)
+    user_info = None
+    if uri.username or uri.password:
+        user_info = uri.username, uri.password
```

Comment:
I think it would be clearer to write this with parentheses?

```python
use_info = (uri.username, uri.password)
```

Reply:
OK, I'll make that change.